.github/workflows/event.yml

```diff
@@ -1,4 +1,7 @@
-name: GitHub Event Handler
+# NOTE: currently azure-sdk-actions only hosts check enforcer code.
+# If further functionality is added, this name should be updated to reflect
+# the more generic behavior
+name: Check Enforcer
```

Comment:
Should we also rename the file to match?

Reply:
I think it's a fair suggestion, but I want people be directed into this file if they try to add new event handlers. Otherwise we may end up having lots of duplicate runners per event (hence why I named it "GitHub Event Handler" originally).